t/lib/t/MusicBrainz/Server/Controller/User/Edit.pm

```diff
@@ -57,13 +51,76 @@ $mech->get_ok($verify_email_path);
 $mech->content_contains("Thank you, your email address has now been verified!");
 
 $mech->get('/user/new_editor');
-$mech->content_contains('http://example.com/~new_editor/');
-$mech->content_contains('hello world!');
 $mech->content_contains('new_email@example.com');
 
 
 };
 
+test 'Limited users cannot edit website and biography' => sub {
+
+my $test = shift;
+my $mech = $test->mech;
+my $c    = $test->c;
+
+MusicBrainz::Server::Test->prepare_test_database($c, '+editor');
+
+$mech->get('/login');
+$mech->submit_form( with_fields => { username => 'new_editor', password => 'password' } );
+
+$mech->get_ok('/account/edit');
+my $tx = test_xpath_html($mech->content);
+html_ok($tx);
+$tx->ok(selector_to_xpath('input#"id-profile.email"'), 'email field for all users');
+$tx->not_ok(selector_to_xpath('input#"id-profile.website"'), 'no website field for limited users');
+$tx->not_ok(selector_to_xpath('input#"id-profile.biography"'), 'no biography field for limited users');
+
+$test->c->sql->do(<<EOSQL, $editor->id);
+    INSERT INTO edit (id, editor, type, status, expire_time, autoedit) VALUES
+        ( 1, 1, 1, $STATUS_APPLIED, now(), 0),
+        ( 2, 1, 1, $STATUS_APPLIED, now(), 0),
+        ( 3, 1, 1, $STATUS_APPLIED, now(), 0),
+        ( 4, 1, 1, $STATUS_APPLIED, now(), 0),
+        ( 5, 1, 1, $STATUS_APPLIED, now(), 0),
+        ( 6, 1, 1, $STATUS_APPLIED, now(), 0),
+        ( 7, 1, 1, $STATUS_APPLIED, now(), 0),
+        ( 8, 1, 1, $STATUS_APPLIED, now(), 0),
+        ( 9, 1, 1, $STATUS_APPLIED, now(), 0),
+        (10, 1, 1, $STATUS_APPLIED, now(), 0);
+EOSQL
+
+$mech->get_ok('/account/edit');
+my $tx = test_xpath_html($mech->content);
```

Comment:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/515/commits/cf46dea1e27697d92d3ea9201419c5c0f2bb165c